elasticapm/metrics/base_metrics.py

```diff
@@ -375,6 +396,46 @@ def val(self, value):
             self._val, self._count = value
 
 
+class Histogram(BaseMetric):
+    DEFAULT_BUCKETS = (0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10, float("inf"))
```

Comment:
@axw what are the default buckets that APM Server creates for transaction duration metrics?

Reply:
Too many to list! APM Server uses HDRHistogram with 2 significant figures by default.